ndn_hydra/repo/modules/global_view.py

```diff
@@ -31,7 +31,8 @@
     size INTEGER NOT NULL,
     origin_node_name TEXT NOT NULL,
     fetch_path TEXT NOT NULL,
-    is_deleted INTEGER NOT NULL DEFAULT 0
+    is_deleted INTEGER NOT NULL DEFAULT 0,
+    last_accessed INTEGER NOT NULL
```

Comment:
Possibly change this `last_accessed` variable to another based on other comment.